snap/snapcraft.yaml

```diff
@@ -0,0 +1,27 @@
+name: scapy
+version: git
+summary: Packet manipulation program.
+description: | A powerful Python-based interactive packet manipulation program and library.
+ 
+grade: devmode
+base: core18
+
+architectures:  
+  - build-on: amd64
+
+parts:
+  scapy:
+    source: https://github.com/secdev/scapy.git
+    source-tag: 2.4.2
+    plugin: python
+    python-version: python3
+
+    stage-packages:
+      - python-six
```

Comment:
Is this required? Scapy does not depend on this package.

Reply:
Hi @guedou, it's not necessary I just add it the package out habit. If you guys do not depend the package I will remove it. :+1:

Reply:
We actually ship it ourselves to have no dependency. Because it's a one-file dependency we allowed this.